### ISSUE TITLE
Request change to Vulkan spec regarding multiViewport feature tied to geometry shaders?

### ISSUE THREAD

**Issue by billhollings** [KhronosGroup/Vulkan-Portability#23]
Metal supports multiple viewports, and MoltenVK enables the `multiViewport` feature, as it [can be useful for `VK_EXT_shader_viewport_index_layer`](https://github.com/KhronosGroup/MoltenVK/issues/1356#issuecomment-834861082).

However, doing so actually violates the Vulkan and SPIR-V specs, which require `multiViewport` to be enabled only if `geometryShader` is enabled, which Metal does not support. CTS enforces that restriction in test `dEQP-VK.info.device_features`.

Per the suggestion by @cdavis5e in the link above, one option might be to request that Vulkan and SPIR-V loosen this general restriction.

**Comment by kvark** [comment]
That sounds good. EIther loosening it upstream, or at least in Vulkan Portability extension itself, makes sense.

**Comment by billhollings** [comment]
>or at least in Vulkan Portability extension itself

Yeah...I contemplated that too. But I don't see any reason why this is portability related per se. Presumably any implementation could benefit by this change.
